### ISSUE TITLE
Strange behavior of get() and image()

### ISSUE THREAD

**Issue by pifragile** [processing/p5.js#5467]
#### Most appropriate sub-area of p5.js?

- [ ] Accessibility (Web Accessibility)
- [ ] Build tools and processes
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Friendly error system
- [x] Image
- [ ] IO (Input/Output)
- [ ] Localization
- [ ] Math
- [ ] Unit Testing
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Other (specify if possible)

#### Details about the bug:

- p5.js version: 1.4.0
- Web browser and version: Chrome 95.0.4638.54
- Operating System: MacOSX
- Steps to reproduce this:

In the following code I would expect a static image with a point at 200,200, because I take an image of the entire canvas and paste it back without any change. But the actual behavior is that the point somehow gets bigger in every iteration of `draw()`.
Reference: https://editor.p5js.org/pifragile/sketches/pdH8Ii5WQ

```
function setup() {
  createCanvas(400, 400);
  stroke(0);
  strokeWeight(30);
  point(200, 200)
}

function draw() {
  let img = get();
  image(img, 0, 0);
}
```

**Comment by golanlevin** [comment]
Just a guess here, but. The repeated ```get()``` / ```image()``` commands effectively sets up a "feedback buffer" in which any minuscule imperfections in graphics rendering propagate and accumulate over time. To see what I mean, compare the outputs of the two programs below, which differ only in whether ```noSmooth()``` is called in Line 5. (An identical difference can be seen if you insert ```pixelDensity(1);``` for Line 5 on your Retina Mac.)

```
function setup() {
  createCanvas(400, 400);
  fill(0);
  noStroke();
  // noSmooth(); // purposefully omitted
  rect(200, 200, 15, 30);
}

function draw() {
  let img = get();
  image(img, 0, 0);
}
```
Compare with:

```
function setup() {
  createCanvas(400, 400);
  fill(0);
  noStroke();
  noSmooth(); // or pixelDensity(1); 
  rect(200, 200, 15, 30);
}

function draw() {
  let img = get();
  image(img, 0, 0);
}
```

It's not that the rectangle gets "bigger" in the first case; rather, a wavefront of (slightly) affected pixels propagates over time; eventually forming a rounded capsule shape! You can suppress this feedback somewhat by putting in a ```background()``` function, but interestingly, you'll still witness some progressive blurring over time, unless you add ```noSmooth()```: 

```
function setup() {
  createCanvas(400, 400);
  background(255);
  fill(0);
  noStroke();
  rect(200, 200, 15, 30);
}

function draw() {
  let img = get();
  image(img, 0, 0);
}
```

**Comment by limzykenneth** [comment]
@pifragile It is as @golanlevin explained above and thus it is expected behaviour. To put in in simpler terms, when drawing ellipse, the edge is usually anti-aliased by having semi transparent pixels to give the impression of being a smooth ellipse. By drawing semi transparent pixels on top of semi transparent pixels as your sketch shows there, those pixels will become fully opaque eventually, giving the impression of becoming bigger (but also rougher).

**Comment by pifragile** [comment]
ok, makes sense. thanks to both of you @golanlevin @limzykenneth !
